index-handlers/src/test/java/no/sikt/nva/nvi/index/IndexDocumentHandlerTest.java

```diff
@@ -183,6 +185,24 @@ void shouldNotFailWhenCandidateIsMissingChannelTypeOrChannelId() {
         assertEquals(expectedIndexDocument, actualIndexDocument);
     }
 
+    @Test
+    void shouldNotFailWhenCreatorIsUnverified() {
+        var institutionId = randomUri();
```

Comment:
Maybe 
```
var request = randomUpsertRequestBuilder()
                          .withUnverifiedCreators(
                              List.of(new UnverifiedNviCreator(randomString(), List.of(randomUri()))))
                          .build();
        Candidate.upsert(request, candidateRepository, periodRepository);
        var candidate = Candidate.fetchByPublicationId(request::publicationId, candidateRepository, periodRepository);
```

To avoid using dao when testing index document generation?